SpeechResponder/Properties/SpeechResponder.resx

```diff
@@ -144,10 +144,6 @@
   <data name="default_is_read_only" xml:space="preserve">
     <value>The default personality cannot be modified. If you wish to make changes, create a copy.</value>
   </data>
-  <data name="default_personality_script_filename" xml:space="preserve">
```

Comment:
Hu what!? Why are we deleting this? I put it in for a reason.

Reply:
Translator entered file names are a potential source for errors. I think it's better to rely on the developer to provide a file that follows the appropriate naming convention.

Reply:
I pre-populated the names on CloudIn, the comment confirms wehat to use, and I audit them.